### PR TITLE
TASK-6: Setup Common UI Library #15

### LINKED ISSUE #15: TASK-6: Setup Common UI Library
**Task Description**

As a developer, I want to setup a UI library so that developers are able to store reusable UI components.

**Acceptance Criteria**

- [ ] Have a UI component library for common components that can be shared across applications

### PR DESCRIPTION
**Pull request by cristian-aldea**
This PR aims to set up a component library for our workspace.

this library will allow us to aggregate all of the common or reusable components we might use across applications. It will also allow us to easily setup storybook in the future.

### CODE REVIEW

File: apps/mull-ui/src/app/app.tsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import './app.scss';
 import { environment } from '../environments/environment';
 import { Message } from '@mull/types';
```
Comment (JimLau49) [inline apps/mull-ui/src/app/app.tsx]: redundant spacing, not sure if this is caused by prettier

**Review by JimLau49 — DISMISSED** [review]
Nice work @TheGreatMarkus  , is the test you wrote able to be run yet, i tested with `nx test` in the root directory but i was unable to run it. But otherwise this looks like a good way to setup our UI lib

Acceptance Criteria

✔ Have a UI component library for common components that can be shared across applications

**Comment by cristian-aldea** [comment]
> Nice work @TheGreatMarkus , is the test you wrote able to be run yet, i tested with `nx test` in the root directory but i was unable to run it. But otherwise this looks like a good way to setup our UI lib
> 
> Acceptance Criteria
> 
> ✔ Have a UI component library for common components that can be shared across applications

To clarify, `nx test` will only run tests on one project (mull-ui), and also I didn't write any tests for this PR, there was just one generated test with this PR.

If you want to run all tests, try `nx run-many --target=test --all`

**Comment by JimLau49** [comment]
> > Nice work @TheGreatMarkus , is the test you wrote able to be run yet, i tested with `nx test` in the root directory but i was unable to run it. But otherwise this looks like a good way to setup our UI lib
> > Acceptance Criteria
> > ✔ Have a UI component library for common components that can be shared across applications
> 
> To clarify, `nx test` will only run tests on one project (mull-ui), and also I didn't write any tests for this PR, there was just one generated test with this PR.
> 
> If you want to run all tests, try `nx run-many --target=test --all`

Ah okay, i was assuming ui-lib.spec.tsx  was written by you 😅

**Comment by cristian-aldea** [comment]
Rebased to be up to date, @JimLau49 feel free to drop a quick reapproval when you can. Thanks for having a look.

**Review by JimLau49 — DISMISSED** [review]
review left above

File: apps/mull-ui/src/app/app.tsx (hunk 1 above)
Comment (r-saba) [reply apps/mull-ui/src/app/app.tsx]: Do you mean the space in `{ environment }` and `{ message }`? That is typically how imports are formatted in JavaScript

**Review by r-saba — DISMISSED** [review]
LGTM!
Ran the generated test and the UiLib component rendered properly on the front-end.

File: apps/mull-ui/src/app/app.tsx (hunk 1 above)
Comment (JimLau49) [reply apps/mull-ui/src/app/app.tsx]: nah line 7

**Review by JimLau49 — APPROVED** [review]
Approved again after fixed space and sonar cube 🎐

**Review by r-saba — APPROVED** [review]
LGTM!
Same review as above. Thanks for dealing with sonarcloud failure.

**Pull request merged**